flamingo/cat_interface.py

```diff
@@ -263,6 +240,12 @@ def compute_cosmo_rs(smiles: pd.Series, opts: Options) -> pd.DataFrame:
         Values
 
     """
-    results = map_reduce(smiles, opts, compute_batch_cosmo_rs, pd.concat)
-
-    return results
+    solvent_dict = {
+        "hexane": "$AMSRESOURCES/ADFCRS/Hexane.coskf",
+        "octadecene": "$AMSRESOURCES/ADFCRS/1-Octadecene.coskf",
+        "toluene": "$AMSRESOURCES/ADFCRS/Toluene.coskf",
+        "o-xylene": "$AMSRESOURCES/ADFCRS/o-Xylene.coskf",
+        "acetonitrile": "$AMSRESOURCES/ADFCRS/acetonitrile.coskf"
+    }
+    run_fast_sigma(smiles.to_list(), solvent_dict)
+    return pd.read_csv("cosmo-rs.csv", header=[0, 1], index_col=0)
```

Comment:
```suggestion
    return run_fast_sigma(smiles, solvent_dict, return_df=True)
```
A few comments here:
* With the `return_df` keyword the relevant dataframe will be returned directly.
* I don't believe the `pd.Series` to `list` conversion here is necessary, right? Since iterating through a series yields its scalar elements.
* The order of the input SMILES is not guaranteed to-be preserved (due to the use of `Pool.imap_unordered`). If this is a problem you may want to sort the index before returning the dataframe.